Libraries/pbxbuild/CMakeLists.txt

```diff
@@ -69,6 +69,6 @@ target_link_libraries(dump_hmap pbxbuild util plist)
 
 ADD_UNIT_GTEST(pbxbuild DirectedGraph Tests/test_DirectedGraph.cpp)
 ADD_UNIT_GTEST(pbxbuild OptionsResolver Tests/test_OptionsResolver.cpp)
-target_link_libraries(test_pbxbuild_OptionsResolver PRIVATE pbxspec pbxsetting plist)
+#target_link_libraries(test_pbxbuild_OptionsResolver PRIVATE pbxspec pbxsetting plist)
```

Comment:
Oops!  This is a bit of leftover hacking from trying to get everything building.  I'll re-enable it.